test/language/expressions/object/object-spread-proxy-ownkeys-no-excluded-keys.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
```

Comment:
2021 ;)

Reply:
Fixed, thanks.

I hope you had a pleasant holiday!